tests/test_m2m.py

```diff
@@ -27,15 +27,15 @@ def test_csv_to_dict(self):
 class MetadataRecordTests(unittest.TestCase):
 
     def setUp(self):
-        xml = b"""<?xml version="1.0" encoding="UTF-8"?>
+        xml = b'''<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Regarding here and other places in the file, for triple quoted strings, I think we tend to use `"""` (three double quote characters) per the [pep8 recommendation](https://www.python.org/dev/peps/pep-0008/#string-quotes).